_posts/2022-11-30-trino-summit-2022-comcast-recap.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Elevating Data Fabric to Data Mesh: solving data needs in hybrid data lakes"
+author: "Sajuman Joseph, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/comcast.jpg
+---
+
+Tune in, for the next [post in the Trino Summit 2022 recap series]({% post_url
+2022-11-21-trino-summit-2022-recap %}). In this post, we're joining "Saj" from
+Comcast, to talk about their migration from a data fabric to data mesh. Saj
+shows you there's more to the buzzword than meets the eye, and gives a solid
+overview of why Comcast is taking data mesh to heart.
+
+<!--more-->
+
+{% youtube sSWBi7bBotQ %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Comcast.pdf">
+  Check out the slides here!
+</a>
+
+## Recap
+
+Comcast engineer, Sajuman Joseph, brings us through the process they went
+through at Comcast to move from their initial use case of using Trino to power
```

Comment:
```suggestion
Comcast engineer Sajuman Joseph brings us through Comcast's process
to move from their initial use case of using Trino to power
```
Wording it this way makes the pronouns a little clearer - it otherwise reads like you're trying to use a singular "they" to refer to Saj.